src/main/java/com/vmware/vipclient/i18n/base/cache/TranslationCacheManager.java

```diff
@@ -28,7 +28,7 @@
     private TranslationCacheManager() {
     }
 
-    public static synchronized TranslationCacheManager createTranslationCacheManager() {
+    public static synchronized TranslationCacheManager getTranslationCacheManager() {
```

Comment:
Is it better rename the method name as 'getInstance'?

Reply:
Made this name change. Please approve, @Xiaochao8 and @huihuiw01 .Thanks!